src/main/docs/guide/toc.yml

```diff
@@ -87,7 +87,12 @@ httpServer:
     title: Configuring the HTTP Server
     threadPools: Configuring Server Thread Pools
     cors: Configuring CORS
-    https: Securing the Server with HTTPS  
+    https: Securing the Server with HTTPS
+  templates:
+    title: Templates
+    thymeleaf: Thymeleaf
+    handlebars: Handlebars.java
```

Comment:
Is the `.java` extension here intentional?

Reply:
Yes, they refer themselves as `handlebars.java` https://jknack.github.io/handlebars.java/